File: Code/05_File_POSIX/Ex6.c
#include <stdio.h>
#include <stdlib.h>
#include <fcntl.h>
#include <unistd.h>

int main(int argc, const char * argv[])
{
	char c;
	int fd = open("exercicio5.txt", O_RDONLY);
	if(fd==-1)
	{
		printf("Erro na abertura"
			" do arquivo.\n");
		exit(1);
	}
	while(read(fd, &c, 1) != 0)
		printf("%c", c);
	close(fd);
	return 0;
}

File: 2_Programação_sistemas_operacionais/2.2_File_POSIX/Python/Ex6.py
import os, stat

try:
	fd = os.open("exercicio5.txt", os.O_RDONLY)
except OSError:
	print("Erro na abertura do arquivo.")
	exit(-1)
s = os.read(fd, 1)
while len(s):
	print(s.decode("utf-8"), end='')
	s = os.read(fd, 1)
os.close(fd)